source/Octopus.Tentacle/Program.cs

```diff
@@ -57,7 +57,12 @@ public override IContainer BuildContainer(StartUpInstanceRequest startUpInstance
             builder.RegisterModule(new ServicesModule());
             builder.RegisterModule(new VersioningModule(GetType().Assembly));
             builder.RegisterModule(new MaintenanceModule());
-            builder.RegisterModule<KubernetesModule>();
+
+            //we don't register any of the kubernetes _stuff_ if we aren't running as a kubernetes agent
+            if (PlatformDetection.Kubernetes.IsRunningAsKubernetesAgent)
```

Comment:
Because the `KubernetesScriptServiceV1Alpha` takes a direct dependency on the Kubernetes code, it was causing k8s services to be registered, where they are expecting to be running in a cluster. This switch means that we only register the `KubernetesModule` when we know we are running as the Kubernetes Agent.